rebar.config

```diff
@@ -13,7 +13,7 @@
   {kvc, ".*",
    {git, "git://github.com/etrepum/kvc.git", {tag, "v1.5.0"}}},
   {riak_kv, ".*",
-   {git, "git://github.com/basho/riak_kv.git", {branch, "develop"}}},
+   {git, "git://github.com/basho/riak_kv.git", {branch, "2.1"}}},
```

Comment:
@jvoegele... needs to stay on riak_kv develop, and we probably need to merge that forward too.